tests/usm/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+file(GLOB test_cases_list *.cpp)
+
+add_cts_test(${test_cases_list})
```

Comment:
USM is a new test category, could you please add it to the standard filter as well?

Reply:
i've applied this comment in https://github.com/KhronosGroup/SYCL-CTS/pull/127/commits/83da3e852fb18ead362208f0f9a64915b65c304c